api/logging/http_info.py

```diff
@@ -0,0 +1,43 @@
+from typing import Union
+
+from flask import Request, Response
+from api import pcm_globals
+
+
+def log_request_body_and_headers(_logger, request: Request):
+    details = __get_http_info(request)
+    details['path'] = request.path
+    if request.args:
+        details['params'] = request.args
+    details['type'] = 'REQUEST'
+    _logger.info(details)
+
+
+def log_response_body_and_headers(_logger, response: Response):
+    details = __get_http_info(response)
+    details['type'] = 'RESPONSE'
```

Comment:
todo: I don't think `type` is needed for both request and response. The same check can be achieved by looking at the availability of the `path` param